src/components/Nav.vue

```diff
@@ -1,28 +1,51 @@
-<script setup>
+<script lang="ts" setup>
+import { computed } from 'vue';
 import { useRoute } from 'vue-router';
 import { useUiStore } from '@/stores/ui';
 
+// TODO: need to import all icons https://github.com/antfu/unplugin-icons/issues/5
+import IHGlobeAlt from '~icons/heroicons-outline/cash';
+import IHNewspaper from '~icons/heroicons-outline/newspaper';
+import IHCash from '~icons/heroicons-outline/cash';
+import IHCog from '~icons/heroicons-outline/cog';
+import IHUsers from '~icons/heroicons-outline/users';
+
 const route = useRoute();
 const uiStore = useUiStore();
 
-const items = {
-  overview: {
-    name: 'Overview'
-  },
-  proposals: {
-    name: 'Proposals'
-  },
-  treasury: {
-    name: 'Treasury'
+const navGroups = {
+  space: {
+    overview: {
+      name: 'Overview',
+      icon: IHGlobeAlt
+    },
+    proposals: {
+      name: 'Proposals',
+      icon: IHNewspaper
+    },
+    treasury: {
+      name: 'Treasury',
+      icon: IHCash
+    },
+    settings: {
+      name: 'Settings',
+      icon: IHCog
+    }
   },
   settings: {
-    name: 'Settings'
+    contacts: {
+      name: 'Contacts',
+      icon: IHUsers
+    }
   }
 };
+const navGroupKey = computed(() => String(route.matched[0]?.name));
+const navGroup = computed(() => navGroups[navGroupKey.value || '']);
```

Comment:
nit. I'd rename it to something more undersandable, maybe `NAVIGATION_CONFIG`, `currentRouteName`, `navigationItems`.